.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
Done. 

-- Roger

> On Mar 19, 2016, at 11:27 PM, Kyle Isom <notifications@github.com> wrote:
> 
> In .gitignore:
> 
> > @@ -0,0 +1,2 @@
> > +.DS_Store
> Can you remove this? It's not needed for this repo, and should probably be in your git core.excludes config.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>